docs/develop/host-specific-api-model.md

```diff
@@ -79,7 +107,11 @@ Excel.run(function (context) {
 
 In the previous example, `selectedRange` is set and its `address` property is loaded when `context.sync()` is called.
 
-Since `sync()` is an asynchronous operation, you should always return the `Promise` to ensure the sync operation completes before the script runs. Doing so ensures that the `sync()` operation completes before the script continues to run. If you're using TypeScript or ES6+ JavaScript, you can `await` the `context.sync()` call instead of returning it. For more information about optimizing performance with `sync()`, see [Excel JavaScript API performance optimization](../excel/performance.md).
+Since `sync()` is an asynchronous operation, you should always return the `Promise` to ensure the sync operation completes before the script runs. Doing so ensures that the `sync()` operation completes before the script continues to run. If you're using TypeScript or ES6+ JavaScript, you can `await` the `context.sync()` call instead of returning it.
```

Comment:
```suggestion
Since `sync()` is an asynchronous operation, you should always return the promise to ensure the `sync()` operation completes before the script continues to run. If you're using TypeScript or ES6+ JavaScript, you can `await` the `context.sync()` call instead of returning the promise.
```

Reply:
I think I'll leave `Promise` as code, since it's referring to the actual object, but I'll take the rest of the suggestion.